From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
Date: Sat, 21 Jan 2012 15:44:52 +0100
Subject: Adapt to libavutil API changes

Add include for libavutil/mathematics.h in ffmpeg_compat.h and writeffmpeg.c
since it is no longer included in libavutil/avutil.h

Thanks: Thomas Preud'homme
---
 intern/ffmpeg/ffmpeg_compat.h                  |    1 +
 source/blender/blenkernel/intern/writeffmpeg.c |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/intern/ffmpeg/ffmpeg_compat.h b/intern/ffmpeg/ffmpeg_compat.h
index e7cb1fe..bb07576 100644
--- a/intern/ffmpeg/ffmpeg_compat.h
+++ b/intern/ffmpeg/ffmpeg_compat.h
@@ -35,6 +35,7 @@
 
 #include <libavcodec/avcodec.h>
 #include <libavutil/rational.h>
+#include <libavutil/mathematics.h>
 
 #if (LIBAVFORMAT_VERSION_MAJOR > 52) || ((LIBAVFORMAT_VERSION_MAJOR >= 52) && (LIBAVFORMAT_VERSION_MINOR >= 101))
 #define FFMPEG_HAVE_PARSE_UTILS 1
diff --git a/source/blender/blenkernel/intern/writeffmpeg.c b/source/blender/blenkernel/intern/writeffmpeg.c
index aa7d71b..2e2633b 100644
--- a/source/blender/blenkernel/intern/writeffmpeg.c
+++ b/source/blender/blenkernel/intern/writeffmpeg.c
@@ -36,6 +36,7 @@
 #include <libavformat/avformat.h>
 #include <libavcodec/avcodec.h>
 #include <libavutil/rational.h>
+#include <libavutil/mathematics.h>
 #include <libswscale/swscale.h>
 #include <libavcodec/opt.h>
 
-- 
